doc/internal-api/local.mk

```diff
@@ -0,0 +1,7 @@
+$(docdir)/internal-api/html/index.html $(docdir)/internal-api/latex: $(d)/doxygen.cfg
```

Comment:
Why not write 
```
ifeq ($(internal_api_docs), yes)
...
else
.PHONY: internal-api-html
internal-api-html:
	@echo "Internal API docs are disabled. Configure with '--enable-internal-api-docs', or avoid calling 'make internal-api-html'."
	@exit 1
endif
```

That way we don't need `mk/disable-internal-api-docs.mk` and the conditional include in `Makefile`.

Reply:
Fixed!